embedded-bins/Makefile.variables

```diff
@@ -1,9 +1,9 @@
-runc_version = 1.0.0-rc93
-containerd_version = 1.4.4
-kubernetes_version = 1.21.0
+runc_version = 1.0.0-rc95
```

Comment:
The PR description says `rc94`, is this right?

Reply:
yeah. when PR was created it was `-rc94`but there is a recent CVE so I added a another commit with `-rc95` in the same PR

Reply:
I updated PR description so it should match